tuning-fork/boilerplate
# Design Component: Button

Closes #176, closes #185

## Overview

* Installs Material UI (for Icons)
* Installs Storybook
* Creates reusable `<Button />` component

The component models the designs on the [figma](https://www.figma.com/file/8KATAkjShf1XANVbgYEIOA/Boilerplate-Component-Library-(based-on-Vaadin-Design-System)?node-id=72%3A174).

## How to test

1. Since this PR installs a few packages, first make sure to `npm install`
2. I've created some stories for the button. To view them, start up storybook by running `npm run storybook`. This should open the storybook in your browser.

## Caveats

* There does not appear to be a particular hover style for outlined contrast buttons. I left the style commented out in the css file for now.
